unit_test/blas/Test_Blas2_gemv.hpp

```diff
@@ -6,12 +6,52 @@
 #include<KokkosKernels_TestUtils.hpp>
 
 namespace Test {
+  template<class ViewTypeA, class ViewTypeX, class ViewTypeY>
```

Comment:
Now that I look at it I think I would be happier if this function can be moved in a test utility header to make it uniform across unit-tests unless we think this might never be used outside of `gemv`?
I had to deal with `vanilla_gemm` when I refactored the unit-tests so I am somewhat sensitive to that topic at the moment x)

Reply:
Done, moved it to TestUtils.